### DISCUSSION TITLE
Too many taps for 2FA

### DISCUSSION THREAD

**Discussion by dragoncoder047** [community/community#109737] [Mobile] [answered]
### Select Topic Area

Product Feedback

### Body

I like using the GitHub mobile app for 2FA…  except that it always involves, what I see, as too many taps.

when I click the button to sign in on a device and get a mobile app notification, I have to tap on the notification to open the app, tap to approve the sign in, then close the app because I’m not using the mobile app, I’m using the other device I just signed into. 

The guys at Cisco figured out how to eliminate the extra taps to close the app with the Duo Mobile app, where the notification already has the sign in details — it says something on the order of “are you trying to sign into &lt;product name>?” — then you tap-and-hold/pull-down the notifications and there are options for approve and deny right on the notification itself. Quick and easy. 

1. Can GitHub implement something like duo’s approach where you don’t have to open the app? (GitHub equivalent: “Are you trying to sign in as &lt;username> on &lt;computer type>?”)
2. If not, why not?

**Comment by stevepopovich** [comment, marked as answer]
Hi @dragoncoder047 thanks for writing in. I have passed this onto the team for consideration.

**Comment by suleymanay** [comment]
dragoncoder047)Yazdığınız için teşekkürler. Bunu değerlendirilmek üzere ekibe ilettim 😍😍

**Comment by suleymanay** [comment]
@dragoncoder047 dragoncoder047)
Yazdığınız için teşekkürler. Bunu değerlendirmek üzere ekibe ilettim 😍😍
